app/scripts/modules/dcos/search/resultFormatter.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.dcos.search.formatter', [])
+  .factory('dcosSearchResultFormatter', function($q) {
+    return {
+      instances: function(entry) {
+        return $q.when((entry.name || entry.instanceId) + ' (' + entry.namespace + ')');
```

Comment:
(don't need to do this as part of this PR, just an FYI)

You can use string templates, which are a little more readable, e.g.
```javascript
`${entry.name || entry.instanceId} (${entry.namespace})`
```